packages/cloud-config/build.yaml

```diff
@@ -7,11 +7,14 @@ requires:
   version: ">=0"
 
 steps:
-- mkdir -p /system /oem
-- cp -rfv oem /system
+- mkdir -p /system/oem
+- cp -rfv oem/{{.Values.oem_file}} /system/oem
 - chmod -R 600 /system/oem
-- cos-tmpl-render /system/oem/00_rootfs.yaml.tmpl
-- cos-tmpl-render /system/oem/06_recovery.yaml.tmpl
+{{ if .Values.templated }}
```

Comment:
I wonder if we should use a helper here to make it simpler...

Reply:
thought about it, but the usage is strictly scoped to this package at the end

Reply:
yes, me too. Probably something like that is enough:

```bash
for tmpl in {{.Values.oem_dir}}/*.tmpl; do
 cos-tmpl-reder "${tmpl}"
done
```
Or, alternatively, as in the above comment it could be applied on per package basis.